scripts/k8s-deployment-sim.sh

```diff
@@ -1,5 +1,20 @@
 #!/bin/bash
 
+printf "\n\n\n*** [0] install server with tls template and HA in k8s\n"
+yq e '.global.agtK8Config.withPlugins.tls.enabled = true' -i k8s/agent+lber/values.yaml
+yq e '.global.agtK8Config.withPlugins.vln.enabled = false' -i k8s/agent+lber/values.yaml
+helm install my-app k8s/agent+lber --set-file global.agtConfig=k8s/example/server+tls.env
+printf "\n\n\n*** [0.1] verify installation\n"
+kubectl get deployments && kubectl get pods && kubectl get services && kubectl get ingress
```

Comment:
add sts checking.

Test requirement:
igs x1
svc x2
pod xlber-replicaCount
pod xagt-replicaCount
deployment x1
sts x1

Reply:
All components are showed in logs